pkg/k8s/k8s.go

```diff
@@ -327,6 +332,75 @@ func StartBackingStore(ctx context.Context, vConfig *config.VirtualClusterConfig
 	return etcdEndpoints, etcdCertificates, nil
 }
 
+func RunCommand(ctx context.Context, command []string, component string) error {
```

Comment:
This is now part of the util/command package I don't think we need this here as well

Reply:
done